tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -46,42 +47,55 @@ public function testGetPackageInfo()
 
     public function testPackagesAreAddedCorrectly()
     {
-        $task = UpdatePackageInfoTask::create();
+        $oldVersionProvider = Injector::inst()->get(VersionProvider::class);
+        try {
+            $task = UpdatePackageInfoTask::create();
```

Comment:
This is to fix a long standing broken unit test when running sink with a forked framework version

The new try/finally block is to reset `Injector::inst()->registerService($versionProvider, VersionProvider::class);` back from the mock VersionProvider, to the original VersionProvder.  Looking through the sapphire teststate code it's not immediately clear to me whether this is required or not (quite possibly it's not and it resets itself between tests), however I don't want to risk causing a regression later on in a subsequent unit test

Looking through the teststate stuff, it's not immediately clear to me whether or not resetting the